src/servers/middlewares/auth-router.ts

```diff
@@ -11,23 +10,22 @@ import { mapError } from "@graphql/error-map"
 import { baseLogger } from "@services/logger"
 import { wrapAsyncToRunInSpan } from "@services/tracing"
 
-export const KratosSdk: (kratosEndpoint?: string) => V0alpha2ApiInterface = (
-  kratosEndpoint,
-) => new V0alpha2Api(new Configuration({ basePath: kratosEndpoint }))
+import { AccountsRepository } from "@services/mongoose"
+import { kratosPublic } from "@services/kratos/private"
```

Comment:
should not import anything under `private` (as the name implies).

Reply:
I didn't really touch this code (expect a function rename). This code is meant to be deleted on Cookie is activated and webwallet updated. (it was a "hack"/proxy for when there was no Oathkeeper that knew how to manage cookie)